pkg/server/api/audit/audit_test.go

```diff
@@ -0,0 +1,283 @@
+package audit_test
+
+import (
+	"errors"
+	"testing"
+
+	"github.com/sirupsen/logrus"
+	"github.com/sirupsen/logrus/hooks/test"
+	"github.com/spiffe/spire-api-sdk/proto/spire/api/types"
+	"github.com/spiffe/spire/pkg/server/api/audit"
+	"github.com/spiffe/spire/test/spiretest"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+func TestEmit(t *testing.T) {
+	log, logHook := test.NewNullLogger()
+
+	for _, tt := range []struct {
+		name       string
+		addFields  logrus.Fields
+		expect     []spiretest.LogEntry
+		emitFields logrus.Fields
+	}{
+		{
+			name: "no fields added",
+			expect: []spiretest.LogEntry{
+				{
+					Level:   logrus.InfoLevel,
+					Message: "Audit log",
+					Data: logrus.Fields{
+						"status": "success",
+						"type":   "audit",
+					},
+				},
+			},
+		},
+		{
+			name: "with fields added",
+			addFields: logrus.Fields{
+				"a": "1",
+				"b": "2",
+			},
+			expect: []spiretest.LogEntry{
+				{
+					Level:   logrus.InfoLevel,
+					Message: "Audit log",
+					Data: logrus.Fields{
+						"status": "success",
+						"type":   "audit",
+						"a":      "1",
+						"b":      "2",
+					},
+				},
+			},
+		},
+		{
+			name: "with fields on emit",
+			emitFields: logrus.Fields{
+				"emit": "test",
+			},
+			expect: []spiretest.LogEntry{
+				{
+					Level:   logrus.InfoLevel,
+					Message: "Audit log",
+					Data: logrus.Fields{
+						"status": "success",
+						"type":   "audit",
+						"emit":   "test",
+					},
+				},
+			},
+		},
+		{
+			name: "with fields on emit and added",
+			addFields: logrus.Fields{
+				"a": "1",
+				"b": "2",
+			},
+			emitFields: logrus.Fields{
+				"emit": "test",
+			},
+			expect: []spiretest.LogEntry{
+				{
+					Level:   logrus.InfoLevel,
+					Message: "Audit log",
+					Data: logrus.Fields{
+						"status": "success",
+						"type":   "audit",
+						"emit":   "test",
+						"a":      "1",
+						"b":      "2",
+					},
+				},
+			},
+		},
+	} {
+		tt := tt
```

Comment:
You are right it is no required, some time ago some lints fails because we was calling a variable from a for inside a function, and it suggest to capture var before, (I think it was golint), 
So I get used to always add it, but for regular `t.Run()` it is not required